README.md

```diff
@@ -371,6 +376,87 @@ Failed: text 'foo' not present on element with locator 'By(css selector, h1)'.
 
 > As you can see, the messages are clear and tell you exactly why the test has failed, such as in the previous example, where a specific text ('foo') is not present in a heading element (an element with css selector 'h1').
 
+## Preparation to next major version
+
+We will show some alerts on console to the user know what changes he need to do.
+
+All changes below is for the library focus only on providing an easy way for software engineers to create robust end-to-end tests.
+
+### Functions that will be removed
+
+1. [getBodyElementFromCurrentBrowserOrBrowserInstance](#getBodyElementFromCurrentBrowserOrBrowserInstance)
+2. [openNewBrowserInTheSamePage](#openNewBrowserInTheSamePage)
+
+#### What to do?
+
+- Remove all functions above to don't have trouble on next major version (v4.0.0).
+
+#### How I know that my code is ok?
+
+You will know that you have solved all your problems when don't see any warning on console like this:
+
+```
+Protractor-helper warning: Function 'getBodyElementFromCurrentBrowserOrBrowserInstance' will be deprecated in version 4.0.0! [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]
+```
+
+### Functions that will be replaced
+
+The 8 below will be replaced to reduce complexity, with the new functions with less arguments and smaller name.
```

Comment:
What about `The 8 below functions will be replaced to reduce complexity and code duplication. Also, they you have shorter names and less arguments.` ?